plugins/woocommerce-blocks/tests/e2e/utils/test.ts

```diff
@@ -168,6 +173,26 @@ const test = base.extend<
 		},
 		{ scope: 'worker', auto: true },
 	],
+	wordPressVersion: async ( {}, use ) => {
+		// Specific the version of WordPress only with x.y format.
+		const latestMinus1Version = 6.6;
+		const stableVersion = 6.7;
+		const nightlyVersion = 6.8;
+		const installedVersion = await getInstalledWordPressVersion();
+
+		if ( installedVersion === latestMinus1Version ) {
+			await use( 'previous' );
+			return;
+		}
+
+		if ( installedVersion === stableVersion ) {
+			await use( 'stable' );
+		}
+
+		if ( installedVersion === nightlyVersion ) {
+			await use( 'nightly' );
+		}
+	},
```

Comment:
I think you're missing a default case when the version is none of the 3 defined.